handlers/handlers.go

```diff
@@ -40,6 +40,15 @@ func read(w http.ResponseWriter, req *http.Request, cfg *config.Config, rend Ren
 	var searchResp searchCli.Response
 	var respErr error
 	var departmentResp searchCli.Department
+
+	if validationProblem {
+		// avoid making any API calls
+		basePage := rend.NewBasePageModel()
+		m := mapper.CreateSearchPage(cfg, req, basePage, validatedQueryParams, []data.Category{}, searchResp, departmentResp, lang, validationProblem)
```

Comment:
Do we need to pass in validateProblem into function? If we do then set this to true

I guess this can be used to determine an error message back in the template for the user to read?